src/main/resources/sqlscripts/CHG5015221_updating.sql

```diff
@@ -0,0 +1,10 @@
+BEGIN;
+
+update cft_task_db.tasks set role_category = 'ADMIN' where role_category is null and jurisdiction = 'CIVIL' and task_name in ('Request for transfer online case');
```

Comment:
Are you planning to run any pre-checks before running this?  
How do you know if the records you expect to revert in the rollback script are the ones you've modified in this line?  You may want to capture the latest numbers just before running the script.  E.g. count total nulls, retrieve all the task_ids etc?

As an example: https://github.com/hmcts/wa-task-management-api/blob/master/src/main/resources/scripts/rwa-2905_update_civil_role_categories.sql

Reply:
You may also want to run some post run verifications

Reply:
Hi Ravi , for pre check I have mentioned that in the CR too I will add those also as script in this PR so that you are assured.

Reply:
I have updated the folder with similar to attached script. Thank you

Reply:
I don't see the scripts you mention above?